dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd

```diff
@@ -54,5 +54,5 @@ publish.wlp.jar.suffix: dev/api/spec
 
 -buildpath: \
 	javax.inject:javax.inject;version=1,\
-	javax.enterprise:cdi-api;version=2.0
-	
\ No newline at end of file
+	javax.enterprise:cdi-api;version=2.0.0.SP1
```

Comment:
Not sure how much of a difference it makes but the original only has one zero
```suggestion
	javax.enterprise:cdi-api;version=2.0.SP1
```

Reply:
This is to comply with bndtools syntax rules